### PR TITLE
(Changed)[Improvement Day] Moved Delegates out of PXOneTapViewController

### PR DESCRIPTION
**Pull request by jscaramal-meli**
Moved delegates out of PXOneTapViewController main class file

### CODE REVIEW

**Review by GiselaARamosC — DISMISSED** [review]

**Review by mvarani — DISMISSED** [review]
LGTM!

File: MercadoPagoSDK/MercadoPagoSDK/Flows/OneTap/UI/PXOneTapViewController/PXOneTapViewController+Delegates.swift [hunk 1]
```diff
@@ -0,0 +1,457 @@
...
+        // Clear installmentsWrapperView
+        installmentsWrapperView.removeAllSubviews()
+
+        guard let installmentData = installmentData, let installmentInfoRow = installmentInfoRow else {
+            return
+        }
+
+        if let selectedCardItem = selectedCard {
+            let properties = self.viewModel.getInstallmentsScreenProperties(installmentData: installmentData, selectedCard: selectedCardItem)
+//            trackScreen(path: TrackingPaths.Screens.OneTap.getOneTapInstallmentsPath(), properties: properties, treatAsViewController: false)
```
Comment (rgrajalesmeli) [inline MercadoPagoSDK/MercadoPagoSDK/Flows/OneTap/UI/PXOneTapViewController/PXOneTapViewController+Delegates.swift:371]: can we remove this commented code? or is it ok to keep that there?

**Review by rgrajalesmeli — APPROVED** [review]
LGTM, just a minor comment

File: MercadoPagoSDK/MercadoPagoSDK/Flows/OneTap/UI/PXOneTapViewController/PXOneTapViewController+Delegates.swift (hunk 1 above)
Comment (jscaramal-meli) [reply MercadoPagoSDK/MercadoPagoSDK/Flows/OneTap/UI/PXOneTapViewController/PXOneTapViewController+Delegates.swift]: Removed, it looks very old and non-sense to be there commented...

**Review by rgrajalesmeli — APPROVED** [review]
LG!!

**Review by GiselaARamosC — APPROVED** [review]

**Pull request merged**